amy/static/calendar_popup.js

```diff
@@ -39,6 +39,7 @@ $(document).ready(function() {
         if (e.date && !$('#id_agreement_end').val()) {
             d = e.date;
             d.setFullYear(d.getFullYear() + 1);
+            d.setDate(d.getDate() - 1);
```

Comment:
@maneesha this one changes date when new membership is created.

When user selects e.g. `2022-05-15` for the agreement start date, then `2023-05-14` will be selected for agreement end date. (Previously it would have been `2023-05-15`.)